scapy/main.py

```diff
@@ -61,7 +61,7 @@
      "the wires and in the waves.", "Jean-Claude Van Damme"),
     ("We are in France, we say Skappee. OK? Merci.", "Sebastien Chabal"),
     ("Wanna support scapy? Star us on GitHub!", "Satoshi Nakamoto"),
-    ("What is dead may never die!", "Python 2"),
+    ("I'll be back.", "Python 2"),
```

Comment:
I will suggest to keep both.

Reply:
Considering we dropped support for Python 2 I'd argue the previous one is out of place.